tasks/bake.js

```diff
@@ -9,7 +9,8 @@ import isValidGlob from 'is-valid-glob';
 
 // modularize manageEnv
 import * as journalize from 'journalize';
-import fs from 'fs';
+
+const config = fs.readJsonSync('./project.config.json');
```

Comment:
From what I read, Node paths are based on where you're calling the function from, so we reference the base directory here.